### PR TITLE
dynapoint: new package

### PR DESCRIPTION
**Pull request by ascob**
Maintainer: me
Compile tested: OpenWRT trunk
Run tested: OpenWRT trunk

Description:
Dynapoint allows dynamic access point creation
and deletion depending on changes of certain network conditions.
More information: https://github.com/thuehn/dynapoint

### CODE REVIEW

**Comment by thess** [comment]
Please remove the LICENSE file - we do not a copy of the GPL with your package.
The PKG_LICENSE_FILES tag is optional and in this case it is not necessary. 

Ditto for https://github.com/openwrt/luci/issues/817

**Comment by ascob** [comment]
I removed the LICENSE files.
I'm not sure what you mean with you second comment, though. There aren't any differences from the linked github-repo except the makefile.

File: net/dynapoint/Makefile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2016 LEDE project (Tobias Ilte)
+#
+# This is free software, licensed under the GNU General Public License v3.
```
Comment (mhei) [inline net/dynapoint/Makefile]: I guess you mixed the license for the Makefile itself and the license
for the payload. The license for the Makefile should be GPLv2, so
I prefer to see here the common header:

# This is free software, licensed under the GNU General Public License v2.
# See /LICENSE for more information.

File: net/dynapoint/Makefile [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dynapoint
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
```
Comment (mhei) [inline net/dynapoint/Makefile]: Please check other packages which do not download external sources but are "self-contained". I think a single version number is sufficient it that case.

File: net/dynapoint/Makefile [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+PKG_MAINTAINER:=Tobias Ilte <tobias.ilte@campus.tu-berlin.de>
```
Comment (mhei) [inline net/dynapoint/Makefile]: Personally, I would prefer some grouping of the variables with empty lines...

File: net/dynapoint/Makefile [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)
+    SECTION:=net
+    CATEGORY:=Network
+    SUBMENU:=wireless
+    DEPENDS:=+lua +libubus-lua +libuci-lua +libubox-lua +luci-lib-nixio
+    MAINTAINER:=Tobias Ilte <tobias.ilte@campus.tu-berlin.de>
```
Comment (mhei) [inline net/dynapoint/Makefile]: Drop this line, you defined already PKG_MAINTAINER variable above.

File: net/dynapoint/Makefile [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+Dynapoint uses LUA scripts to allow dynamic access point creation
+and deletion depending on changes of certain network conditions.
+endef
+
+define Package/$(PKG_NAME)/conffiles
+/etc/config/dynapoint
+endef
+
+define Build/Compile
+	true
```
Comment (mhei) [inline net/dynapoint/Makefile]: I think you can remove the line and use an empty body here.

File: net/dynapoint/Makefile [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    SUBMENU:=wireless
+    DEPENDS:=+lua +libubus-lua +libuci-lua +libubox-lua +luci-lib-nixio
+    MAINTAINER:=Tobias Ilte <tobias.ilte@campus.tu-berlin.de>
+    TITLE:=Dynamic access point manager
+endef
+
+define Package/$(PKG_NAME)/description
+Dynapoint uses LUA scripts to allow dynamic access point creation
+and deletion depending on changes of certain network conditions.
+endef
```
Comment (mhei) [inline net/dynapoint/Makefile]: Typically, two whitespaces are in front of the description on each line.

File: net/dynapoint/src/dynapoint.lua [hunk 7]
```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/lua
+
```
Comment (mhei) [inline net/dynapoint/src/dynapoint.lua]: I'd add some copyright and license notes here.

**Comment by ascob** [comment]
Done. Thanks.

**Comment by ascob** [comment]
Anything else which needs to be done?

**Comment by thuehn** [comment]
run time tested on 6 APs in my office network, works fine & so far no issues.
Acked-by: Thomas Huehn <thomas.huehn@evernet-eg.de>

**Comment by ascob** [comment]
Ping @mhei @thess

**Pull request closed without merging**